umbraco/Umbraco-CMS
# Feature indicate custom crop

### Prerequisites

- [x] I have [created an issue](https://github.com/umbraco/Umbraco-CMS/issues) for the proposed changes in this PR, the link is: https://github.com/umbraco/Umbraco-CMS/issues/2932

### Description
This PR adds a visual indication if an image crop is overridden, as explained/requested in #2932.

To test:

1. Make sure you have defined crops on the Image Cropper data type for the Media item
2. Upload a new image
3. Edit one of the crops and save the edit
4. Notice the little green indicator in the upper right corner of the edited crop
5. Reset the edited crop
6. Ensure that the indicator isn't visible anymore

![image cropper custom crop indicator](https://user-images.githubusercontent.com/7405322/46454436-66eb5f00-c7a6-11e8-8734-3872bdd53dbe.gif)


**Note:** The indicator shows immediately after you've opened a crop for editing. That's because the moment you edit one, a custom crop is created. Even if you don't do any editing to the crop, it's saved and disconnected from the main image (unless you choose "reset" in the crop editor). I'm going to have a look at improving this UX a bit in another PR 😄
